### ISSUE TITLE
[Adaptive Cards - Designer] - Screen reader is not announcing state information as 'Expand/Collapsed' for the 'Bind...' and 'Add an action' expandable control on the page.

### ISSUE THREAD

**Issue by Rishikesh6521** [microsoft/AdaptiveCards#9312]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/
OS Version: 25H2 (OS Build 26200.5761)
Browser Version: Version 145.0.3734.1 (Official build) dev (64-bit)
Screen Reader: Narrator, NVDA

**Repro Steps:**

1. Turn on Narrator through Ctrl + Windows + Enter key.
2. Open the above URL on the Edge Dev browser.
3. Navigate to the 'Designer' control present on the header section of the page using tab key and activate using enter key.
4. Navigate to the 'Bind...' and 'Add an action' expandable control on the 'Adaptive Card editor' section using tab key and activate using enter key.
5. Verify that, screen reader is announcing state information as 'Expand/Collapsed' for the 'Bind...' and 'Add an action' expandable control on the page or not.

**Actual Result:**
Screen reader is not announcing state information as 'Expand/Collapsed' for the 'Bind...' and 'Add an action' expandable control on the page.

**Observation:**
Same issue replicates with NVDA

**Note:**
Same issue replicates for the 'Add an action' expandable control on the page present inside the 'ELEMENT PROPERTIES' flyout.

**Expected Result:**
Screen reader should properly announce state information as 'Expand/Collapsed' for the 'Bind...' and 'Add an action' expandable control on the page.

**User Experience:**
Visually impaired users who rely on screen reader to access the web content will be impacted if screen reader is not announcing state information as 'Expand/Collapsed' for the 'Bind...' and 'Add an action' expandable control on the page due to which user will not be able to know the purpose, functionality and current state of the control if it is not announced by screen reader.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/user-attachments/assets/0cf569b3-14e3-4c65-a857-0ffe48ad7f69

![Image](https://github.com/user-attachments/assets/fdaa3f84-d235-4508-a23b-2706d688db54)

### Card JSON

```JSON
NA
```

**Comment by Rishikesh6521** [comment]
Designer page is going to be deprecated, hence closing the bug.
